src/screens/OtherProfile/index.js

```diff
@@ -179,76 +166,10 @@ const OtherProfile = () => {
   }, [other_id]);
 
   const initData = () => {
-    setInitLoading(true);
     setUserId(params.data.user_id);
     setUsername(params.data.username);
-    fetchOtherProfile();
-  };
-
-  const checkUserBlockHandle = async (userId, callback) => {
-    const status = await netInfo.fetch();
-    if (status.isConnected) {
-      try {
-        const sendData = {
-          user_id: userId
-        };
-        const processGetBlock = await checkUserBlock(sendData);
-        if (callback) callback();
-        if (processGetBlock.status === 200) {
-          setBlockStatus(processGetBlock.data.data);
-        }
-      } catch (e) {
-        if (__DEV__) {
-          console.log(e, 'error');
-        }
-      }
-    }
   };
 
-  const handleSaveDataOtherProfile = (data) => {
-    setDataMain(data);
-    setDataMainBio(data.bio);
-    checkUserBlockHandle(data.user_id);
-    setOtherId(data.user_id);
-    setLoading(false);
-    setInitLoading(false);
-  };
-
-  const fetchOtherProfile = async () => {
-    const status = await netInfo.fetch();
-    if (status.isConnected) {
-      try {
-        handleOfflineMode();
-        const result = await getOtherProfile(params?.data?.username);
-        if (result.code === 200) {
-          handleSaveDataOtherProfile(result.data);
-        }
-        StorageUtils.otherProfileData.setForKey(
-          params?.data?.username,
-          JSON.stringify(result.data)
-        );
-      } catch (e) {
-        if (e.response && e.response.data && e.response.data.message) {
-          SimpleToast.show(e.response.data.message, SimpleToast.SHORT);
-        }
-        setLoading(false);
-        setInitLoading(false);
-      }
-    } else {
-      handleOfflineMode();
-    }
-  };
-
-  const handleOfflineMode = async () => {
-    const cache = await StorageUtils.otherProfileData.getForKey(params?.data?.username);
-    if (cache) {
-      const data = JSON.parse(cache);
-      handleSaveDataOtherProfile(data);
-    } else {
-      setLoading(false);
-      setInitLoading(false);
-    }
-  };
   const onShare = async () => ShareUtils.shareUserLink(username);
 
   const handleSetUnFollow = async () => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `initData` has been simplified, which is good for readability. However, the removal of the `fetchOtherProfile` function call from `initData` might affect the functionality if it's not called elsewhere. Make sure that the profile data is still fetched correctly after this change.



<!-- This is an auto-generated comment by OSS CodeRabbit -->